budgie.adoc

```diff
@@ -0,0 +1,36 @@
+=== Budgie desktop on FreeBSD
+
+Links: +
+link:https://blog.buddiesofbudgie.org/[Buddies of Budgie news] URL: link:https://blog.buddiesofbudgie.org/[https://blog.buddiesofbudgie.org/]
+link:https://codeberg.org/olivierd/freebsd-ports-budgie[Development repository] URL: link:https://codeberg.org/olivierd/freebsd-ports-budgie[https://codeberg.org/olivierd/freebsd-ports-budgie] +
+
+Contact: Olivier Duchateau <duchateau.olivier@gmail.com>
+
+Budgie initially developed as the default desktop environment for the former Evolve OS. Since the 10.6.x releases, improvements have been made to be "agnostic".
+
+It is build on top of the GNOME technologies such as GTK >= 3, GLib, Mutter, libpeas.
+
+The goal is to have a new desktop for end-users. I have submitted 2 reviews (link:https://reviews.freebsd.org/D37224[D37224] and link:https://reviews.freebsd.org/D37286[D37286] for Porter Handbook) in order committers import it.
+
+We can find:
+
+* Mk/Uses framework `budgie.mk`
+* new virtual category (*budgie*)
+* 6 applications
+* icon theme `x11-themes/tela-icon-theme`
+
+During this quarter, I have also submitted several patches (related to this desktop) especially:
+
+* `x11/gnome-terminal` update to 3.44.2 link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=267928[bug #267928]
+* `x11-wm/mutter` update to 42.6 link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=267899[bug #267899]
+* `x11-toolkits/libwnck3` update to 40.1 link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=267898[bug #267898]
+
+These bugs are also still open:
+
+* `devel/libpeas` link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=267420[bug #267420]
+* `sysutils/gnome-settings-daemon` link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=265107[bug #265107]
+
+==== Open tasks
+
+* Re-organize some sections in Handbook
+* Migrate my link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=265107[documentation] into Handbook
```

Comment:
Please clarify: is that the FreeBSD Handbook, or a separate handbook for Budgie itself?

Reply:
I would like to add support of LightDM as XDM in the Handbook. This is default login manager with Xfce 4.16, and also in this desktop. Users come from Linux already use it, but with FreeBSD we must to adjust (or create) configuration files. Most of them are common to all desktops environment.

.xinitrc file is only used for xdm or without login manager. LightDM loads .xprofile (which I provide in meta-port `x11-wm/budgie`). This file ensures that environment variables (used by the desktop) are defined, this avoids warnings in .xsessions-error or in the worst case core dump.

About migration of my notes of Budgie desktop, I was thinking rather to FreeBSD's wiki (something more official). But I can maintain them on my side.